[Internal][ResponseOps][Alerts]: Add workaround wording to access stack alerts page in solution view (#4310)
elastic/docs-content: explore-analyze/alerts-cases/alerts/view-alerts.md

When the conditions of a rule are met, it creates an alert. If the rule has actions, they run at the defined frequency. For example, the rule can send email notifications for each alert at a custom interval. For an introduction to the concepts of rules, alerts, and actions, refer to [Alerting](../alerts.md).

Manage alerts from the following places:

> - **Rule details page**: On a rule's details page, alerts generated by the rule are displayed in the Alerts table. To access the rule's details page, find the **Rules** management page in the navigation menu or using the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md) (**Alerts and Insights > Rules**), then click the rule's name.

Author: Small tweak here:
```suggestion
- **Rule details page**: On a rule's details page, alerts the rule generates appear in the Alerts table. To access a rule's details page, find the **Rules** management page in the navigation menu or using the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md) (**Alerts and Insights > Rules**), then click its name.
```

When the conditions of a rule are met, it creates an alert. If the rule has actions, they run at the defined frequency. For example, the rule can send email notifications for each alert at a custom interval. For an introduction to the concepts of rules, alerts, and actions, refer to [Alerting](../alerts.md).

Manage alerts from the following places:

- **Rule details page**: On a rule's details page, alerts the rule generates appear in the Alerts table. To access a rule's details page, find the **Rules** management page in the navigation menu or using the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md) (**Alerts and Insights > Rules**), then click its name.
- **Alerts management page**: Alerts generated by {{stack-manage-app}} rules appear on the **Alerts** management page. Accessing this page differs based on the [navigation view](../../../deploy-manage/manage-spaces.md#spaces-managing) that you're using:

>     -  **Using Classic navigation**: Find the **Alerts** management page in the navigation menu or use the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md) (**Stack Management > Alerts**). 

Reviewer: ```suggestion
    -  **Classic navigation**: Find the **Alerts** management page in the navigation menu or use the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md) (**Stack Management > Alerts**). 
```
